deploy/cr.yaml

```diff
@@ -10,7 +10,7 @@ spec:
 #  pause: true
 #  unmanaged: false
   crVersion: 1.11.0
-  image: percona/percona-server-mongodb:4.4.8-9
+  image: percona/percona-server-mongodb:4.4.9-10
   imagePullPolicy: Always
```

Comment:
please update the CR (add all new options for all needed section)

Reply:
d14d72e